fastly/fastly_test.go

```diff
@@ -290,6 +290,68 @@ wMfrTEOvx0NxUM3rpaCgEmuWbB1G1Hu371oyr4srrr+N
 `
 }
 
+// pgpPublicKeyUpdate returns a PEM encoded PGP public key suitable for testing
+// updates to the public key field.
+func pgpPublicKeyUpdate() string {
```

Comment:
I'm "iffy" on doing this.

Reply:
Since there are fake keys all over the test suite, this doesn't bother me _too_ much. Worth adding to the list of testing cleanup TODOs, though.

Reply:
What are your concerns?